### PR TITLE
[TST] Add flake8-black to test requirements

### LINKED ISSUE #298: Include black formatting evaluation in style check
I'd like to look into the [black code formatter GitHub Action](https://github.com/marketplace/actions/black-code-formatter). @akimbler is planning to test this out with his own repos, so we'll wait on what he has to say.

### PR DESCRIPTION
**Pull request by akimbler**
Closes #298 
* Adds `flake8-black` to test requirements.
* Adds Adam Kimbler to Zenodo file.

### CODE REVIEW

**Comment by tsalo** [comment]
@akimbler Feel free to add a `pyproject.toml` with black config info in this PR. I don't think I'll get #302 working any time soon.

**Comment by tsalo** [comment]
Okay I added `pyproject.toml` with #303. I decided to drop the `nimare/info.py` --> `setup.cfg` migration, since that's what was causing #302 to break.

File: setup.cfg [hunk 1]
```diff
@@ -15,3 +15,12 @@ exclude = *build/,info.py,_version.py,due.py
 putty-ignore =
     */__init__.py : +F401
 ignore = E203,E402,E722,W503
+
+[options.extras_require]
```
Comment (tsalo) [inline setup.cfg:19]: you can drop this section. `setup.cfg` is my enemy now.

Comment (akimbler) [reply setup.cfg]: Done

**Comment by tsalo** [comment]
@akimbler do you want to try just updating the flake8 settings to match black's recommendations? Like, ignore W503 but not W504, etc.?

**Comment by akimbler** [comment]
@tsalo Where are the recommendations? We already ignore W503 (Line Break Before Operator) and E203 (No Space Before ':') which are the two I'm most familiar with

**Comment by tsalo** [comment]
If you swap out flake8 with flake8-black in the style check CI job, what does it do again? Does it say _what_ black has a problem with, or just that "black would change this"?

**Comment by akimbler** [comment]
BLK100 (Black would change this) is the only error flake8-black throws other than errors with black installation

**Comment by tsalo** [comment]
I really don't like that. It's so uninformative that I think it would only make things worse. At least if it's run as a replacement for flake8. How about adding a flake8-black job to the CI? So we still have flake8 (and reduce our ignored warnings/errors to better match black) and we also run flake8-black. That way, if a PR passes flake8 but fails flake8-black, they will at least know that it's a more nuanced issue to be handled.

**Comment by akimbler** [comment]
> I really don't like that. It's so uninformative that I think it would only make things worse. At least if it's run as a replacement for flake8. How about adding a flake8-black job to the CI? So we still have flake8 (and reduce our ignored warnings/errors to better match black) and we also run flake8-black. That way, if a PR passes flake8 but fails flake8-black, they will at least know that it's a more nuanced issue to be handled.

flake8-black isn't executable, it's just an additional set of tests for `flake`. What we could do is have a separate check with `flake8 --select BLK100` to limit the report to only `black` errors

**Comment by tsalo** [comment]
Ohhh okay. So flake8-black _adds_ BLK100 to the existing flake8 checks? It doesn't replace them? E.g., if there's a W504, flake8 will still say W504, not BLK100.

**Comment by tsalo** [comment]
Can you commit a couple of style mistakes so I can see how it will look? We can just revert them afterwards and the commit won't show up since we squash and merge.

**Comment by akimbler** [comment]
Yes, my attempt at making a single W504 worked

**Comment by tsalo** [comment]
Do you know what the BLK100 at 127 corresponds to?

**Comment by tsalo** [comment]
Regardless, feel free to revert that change and I'll merge after. Thanks for doing this!

**Comment by akimbler** [comment]
> Do you know what the BLK100 at 127 corresponds to?

It's specifically referring to the W504 error, it does the PEP8 supported break before the operator

**Pull request merged**